docs/pguidance/agent-install-awsecs.md

```diff
@@ -0,0 +1,123 @@
+# Install to AWS EC2
+
+Deploy the SignalFx Smart Agent to an AWS EC2 instance using a SignalFx
+configuration script, and run the Smart Agent as a Daemon service in an
+EC2 cluster.
+
+## Prerequisites
+
+* Access to the Amazon Web Services (AWS) Elastic Compute Cloud
+  (EC2) web console or the AWS Command Line Interface (CLI). To learn more,
+  refer to the AWS EC2 documentation.
+* SignalFx access token. See [Smart Agent Access Token](https://docs.signalfx.com/en/latest/integrations/agent/smart-agent-access-token.html).
+* Your SignalFx realm. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+
+## Configure the Smart Agent for AWS EC2
+
+Configure the Smart Agent for AWS EC2 by following these steps:
```

Comment:
Per the editors, we should always use the full product name. So, "SignalFx Smart Agent."